llvm/include/llvm/CodeGen/LiveRegMatrix.h

```diff
@@ -59,15 +59,11 @@ class LiveRegMatrix {
   unsigned RegMaskVirtReg = 0;
   BitVector RegMaskUsable;
 
-  LiveRegMatrix() = default;
+  LiveRegMatrix() : LIUAlloc(new LiveIntervalUnion::Allocator()) {};
```

Comment:
nit: could also do `LIUAlloc(std::make_unique<LiveIntervalUnion::Allocator>)` up to you.